bugbug/bugzilla.py

```diff
@@ -104,7 +104,9 @@ def bughandler(bug):
     return all_ids
 
 
-def get(ids_or_query):
+def get(
+    ids_or_query, download_comment=True, download_attachment=True, download_history=True
```

Comment:
Nit: rename the parameters `get_comment` or `comment` instead of `download_comment` (same with attachment and history)